sqllogictest-bin/Cargo.toml

```diff
@@ -34,5 +34,5 @@ tokio = { version = "1", features = [
     "process",
 ] }
 fs-err = "2.9.0"
-tracing-subscriber= "0.3" 
-tracing = "0.1"
\ No newline at end of file
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
```

Comment:
When `"env-filter"` is not enabled, `RUST_LOG` can also work, but default level is `INFO`...